README.md

```diff
@@ -1,2 +1,65 @@
 # bomctl
-Format agnostic SBOM tooling
+
+__bomctl__ is format-agnostic Software Bill of Materials (SBOM) tooling, which is intended to bridge the gap between SBOM generation and SBOM analysis tools. It focuses on supporting more complex SBOM operations by being opinionated on only supporting the [NTIA minimum fields](https://www.ntia.doc.gov/files/ntia/publications/sbom_minimum_elements_report.pdf) or other fields supported by [protobom](https://github.com/bom-squad/protobom).
+
+__This is an experimental project under active development. We'd love feedback on the concept, scope, and architecture!__
```

Comment:
```suggestion
> [!NOTE]
> This is an experimental project under active development. We'd love feedback on the concept, scope, and architecture!
```

Reply:
See [GitHub markdown alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)